cmd/norduser/main.go

```diff
@@ -90,14 +90,21 @@ func startSnap() {
 		os.Exit(int(childprocess.CodeFailedToEnable))
 	}
 
-	configDirPath, err := internal.GetConfigDirPath(usr.HomeDir)
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
```

Comment:
if not message can be logged, shouldn't be different error codes for this errors? otherwise might be hard for customer support and user to identify the error.